environments/icds-cas/public.yml

```diff
@@ -32,7 +32,6 @@ redis_maxmemory_policy: allkeys-lru
 redis_auto_aof_rewrite_percentage: "50"
 
 kafka_log_dir: '{{ encrypted_root }}/kafka'
-kafka_log_retention: 1440 # 60 days
```

Comment:
What's the default value for this when it's not set? Is there a danger of not being able to reprocess few days old kafka changes?

Reply:
The default value is 28 days https://github.com/dimagi/commcare-cloud/pull/2216/files#diff-8d03e4e8b3ffdde0debe5a39f6edbea8R9

I moved it to 60 days as an emergency measure when we were having speed issues with the original object storage